tests/run_sources_tests.py

```diff
@@ -28,8 +28,10 @@ def main(argv):
     for s in os.listdir(src_dir):
         if os.path.splitext(s)[1] in ('.cpp', '.h'):
             t = tu.SourceTest()
-            t.build(test_input=os.path.join(src_dir, s),
-                    test_lang='CPP', test_config=config)
+            filepath = os.path.join(src_dir, s)
+            t.build(test_input=filepath,
```

Comment:
(Nit: saving a line and putting `test_input` and `test_expected` on the same line would be nice :slightly_smiling_face:.)